### PR TITLE
To do list list structure

### PR DESCRIPTION
**Pull request by rega1237**
# In this branch added:

- [ ] Setup the repo with Webpack
- [ ] Webpack and work src files to compile in the dist folder
- [ ] Add structure for the to-do list
- [ ] Add dynamically 3 elements to the list with javascript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+- Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/rega1237/to-do-list).
+2. Press the "Code" button and copy the link.
+3. Clone it using git command `git clone <link>`.
+
```
Comment (andersonlebon) [inline README.md:20]: - [ ] Kindly provide instructions in your Readme on how to install dependencies of the project(`npm install`) as well as running it on the local server(`npm start`).

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  {
+    description: 'fix the car',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'walking the dog',
+    completed: false,
+    index: 0,
+  }];
```
Comment (andersonlebon) [inline src/index.js:18]: ## [OPTIONAL]

- [ ] Kindly make sure your indexes are unique as this should uniquely identify the to-do task.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @rega1237 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
## Hi @rega1237, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**